tests/test_serialization.py

```diff
@@ -44,8 +44,10 @@ def test_bytes(self):
       self.assertJson(b'\x01\x02\x03', '{"@bytes":"AQID"}')
 
   def test_query(self):
-    self.assertJson(Query({"lambda": "x", "expr": {"var": "x"}}),
-                    '{"@query":{"expr":{"var":"x"},"lambda":"x"}}')
+    expected1 = '{"@query":{"expr":{"var":"x"},"lambda":"x"}}'
+    self.assertJson(Query({"lambda": "x", "expr": {"var": "x"}}), expected1)
+    self.assertJson(Query({"api_version": "3", "lambda": "x", "expr": {"var": "x"}}),
+                    '{"@query":{"api_version":"3","expr":{"var":"x"},"lambda":"x"}}')
```

Comment:
a minor thing here: are you going to extract the expected string to `expected2`? or inline both asserts?  (I don't mind, just asking for consistency)